lib/pdk/module/update.rb

```diff
@@ -6,6 +6,8 @@ class Update < Convert
       GIT_DESCRIBE_PATTERN = %r{\A(?<base>.+?)-(?<additional_commits>\d+)-g(?<sha>.+)\Z}
 
       def run
+        template_uri.git_ref = new_template_version
```

Comment:
When you run a `pdk update` on a module that was generated via a tag, the update will use `master` as the default template_ref. Is this expected? Do we expect the UX to be that the user has to specify the `--template-ref` every time?

Reply:
If you're running PDK from a gem install or local checkout, yeah it'll default to `master` (which is the same as the current behaviour). If you're running from a packaged install, it'll default to the package version as before, so `pdk update` on 1.9.1 would be the equivalent of `pdk update --template-ref 1.9.1`.

An alternative would be that we change the behaviour for "development mode" so that instead of `master` we inspect the template repo and find the highest tag number to use. This would be a behaviour change though and would require a major version bump.

Reply:
In my case, I ran `pdk new module foo --template-url="https://github.com/puppetlabs/pdk" --template-ref="1.7.0"` and it created the module fine, but then when I ran `pdk update` it ran the update against `master`.

Reply:
As a user I'd expect the choice of template ref to stick here

Reply:
This appears to be fixed by the latest update.